vue.config.js

```diff
@@ -19,7 +19,7 @@ module.exports = defineConfig({
     }
   },
   pwa: {
-    themeColor: '#2196F3',
+    themeColor: '',
```

Comment:
@BastelKlug was there any specific reason for this line to change?

I ask this as I've noticed this warning on the console caused by it:

![image](https://user-images.githubusercontent.com/85504/181502200-1ce55a31-47b3-4017-b5e6-a1349b25ff0e.png)

Reply:
Yes, when I removed the line, it took the default value from the plugin. So I left it blank. Maybe there is another way 🤔

Reply:
MS Edge in Win11 seems to work fine with the value in... What browser/phone did you notice that you needed to clear that? Might be the case here...

Reply:
Chrome on Windows 11. Maybe it was a bug with the Vue Cli Hot reloading?

Reply:
Chrome on MacOS, also does not work:

https://user-images.githubusercontent.com/44060099/181530634-372550b6-c623-44e8-ae4b-c375b877c059.mov

And also the bar in Safari that you see when you scroll up too high:
<img width="1440" alt="Bildschirmfoto 2022-07-28 um 16 17 19" src="https://user-images.githubusercontent.com/44060099/181532958-0a8a3356-cc8e-4be2-994f-7862af3981e7.png">

Reply:
Ah, I just tested with Chrome on Win11 and worked fine... guess this is a Mac thing only then?